src/storage/src/controller/hosts.rs

```diff
@@ -291,7 +274,7 @@ where
     }
 
     /// Stops an orchestrated storage host for the specified ID.
```

Comment:
```suggestion
    /// Drops an orchestrated storage host for the specified ID.
```
Just to keep parity with the new fn name.